CMake build error when build with flag 'BUILD_WITH_CODEGEN_SUPPORT'

## Bug description

When trying to build the repository from source with flag 'BUILD_WITH_CODEGEN_SUPPORT', I encounter the error:
CMake Error at cmake_install.cmake:62 (file):
  file INSTALL cannot find
  "/home/tkpf//crocoddyl_v3/cmake/find-external/cppad": No such file
  or directory.

It seems like this is an error because of the lowercase spelling. 
Changing line: https://github.com/loco-3d/crocoddyl/blob/devel/CMakeLists.txt#L199
to 
*add_project_dependency(cppad 20200000.0 REQUIRED FIND_EXTERNAL CppAD)*
fixed it for me.


### System

- OS: Ubuntu 22.04.5 
- Crocoddyl version: 3.0.1

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -196,7 +196,7 @@ endif()
 if(BUILD_WITH_CODEGEN_SUPPORT)
   set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/find-external/CppAD/"
                         ${CMAKE_MODULE_PATH})
-  add_project_dependency(cppad 20200000.0 REQUIRED FIND_EXTERNAL cppad)
+  add_project_dependency(cppad 20200000.0 REQUIRED FIND_EXTERNAL CppAD)
   add_project_dependency(cppadcg 2.4.1 REQUIRED)
   add_project_dependency(pycppad 1.2.4 REQUIRED)
   add_definitions(-DCROCODDYL_WITH_CODEGEN)

--- a/notebooks/05_codegenerating_a_cartpole.ipynb
+++ b/notebooks/05_codegenerating_a_cartpole.ipynb
@@ -390,8 +390,8 @@
     "ration_cg = t_calc_ad / t_calc\n",
     "ration = t_calcDiff_ad / t_calcDiff\n",
     "print()\n",
-    "print(f\"Code generation with analytical derivatives is {ration_cg:.2f} times than AutoDiff.\")\n",
-    "print(f\"Analytical derivatives are {ration:.2f} times than AutoDiff.\")"
+    "print(f\"Code generation with analytical derivatives is {ration_cg:.2f} times as fast as AutoDiff.\")\n",
+    "print(f\"Analytical derivatives are {ration:.2f} times as fast as AutoDiff.\")"
    ]
   },
   {

